app/Platform/Actions/BuildGameShowPagePropsAction.php

```diff
@@ -293,18 +293,67 @@ private function buildAggregateCredits(Game $game): AggregateAchievementSetCredi
             ->values()
             ->all();
 
+        // Get the sorted author credits to use for filtering.
+        $sortedAuthors = $sortByCountDesc($achievementsAuthors, true);
+
+        // Filter logic credits to exclude authors - matching the client-side logic.
+        $authorUserIds = $achievementsAuthors->keys()->toArray();
+        $filteredLogicCredits = $achievementsLogicCredits->except($authorUserIds);
+
+        // Deduplicate artwork credits (achievement + badge artwork) and exclude authors.
+        $deduplicatedArtworkCredits = $this->deduplicateCredits([
+            $achievementSetArtworkCredits,
+            $achievementsArtworkCredits,
+        ])->except($authorUserIds);
```

Comment:
Something about this logic is cloning the game badge credit into achievement credit.
http://localhost:64000/game2/32722

master:
<img width="375" height="130" alt="image" src="https://github.com/user-attachments/assets/6c02ed35-9b23-4cf3-bf52-9a80f2a34e65" />

this branch:
<img width="379" height="153" alt="image" src="https://github.com/user-attachments/assets/c0bdb647-f329-4a4e-8967-1ecb10548c81" />